OpenSim/ExampleComponents/RegisterTypes_osimExampleComponents.cpp

```diff
@@ -0,0 +1,65 @@
+/* -------------------------------------------------------------------------- *
+ *             OpenSim:  RegisterTypes_osimExampleComponents.cpp              *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2017 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include <OpenSim/Common/Object.h>
+#include "RegisterTypes_osimExampleComponents.h"
+
+#include "ToyReflexController.h"
+
+#include <string>
+#include <iostream>
+#include <exception>
+
+using namespace std;
+using namespace OpenSim;
+
+static osimSimulationInstantiator instantiator; 
+
+//_____________________________________________________________________________
+/**
+ * The purpose of this routine is to register all class types exported by
+ * the osimSimulation library.
```

Comment:
Isn't this PR about creating a new osimExampleComponents library, or are you adding these example classes to the existing osimSimulation library?